### PR TITLE
Add `os` argument to `cce_node_v3`

### LINKED ISSUE #712: OS should be configurable for CCE node
As per documentation:
```
Node OS.

    Nodes in clusters of Kubernetes v1.11 or earlier support EulerOS 2.2.
    Nodes in clusters of Kubernetes v1.13 or later support EulerOS 2.5.
    Nodes in clusters of Kubernetes v1.15 support EulerOS 2.5.
    Nodes in clusters of Kubernetes v1.17 support EulerOS 2.5 and CentOS 7.7.
```

Reference: https://docs.otc.t-systems.com/en-us/api2/cce/cce_02_0242.html

### PR DESCRIPTION
**Pull request by outcatcher**
## Summary of the Pull Request
Resolve #712

## PR Checklist

* [x] Refers to: #712
* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

New:
```
=== RUN   TestAccCCENodesV3_os
--- PASS: TestAccCCENodesV3_os (778.32s)
PASS

Process finished with exit code 0
```
Regression:
```
=== RUN   TestAccCCENodesV3_basic
--- PASS: TestAccCCENodesV3_basic (714.44s)
PASS

Process finished with exit code 0
```

### CODE REVIEW

**Review by anton-sidelnikov — DISMISSED** [review]

File: docs/resources/cce_node_v3.md [hunk 1]
```diff
@@ -40,6 +41,16 @@ The following arguments are supported:
...
 * `cluster_id` - (Required) ID of the cluster. Changing this parameter will create a new resource.
 
+* `flavor_id` - (Required) Specifies the flavor id. Changing this parameter will create a new resource.
+
+* `availability_zone` - (Required) specify the name of the available partition (AZ). Changing this parameter will create a new resource.
+
+* `key_pair` - (Required) Key pair name when logging in to select the key pair mode. Changing this parameter will create a new resource.
+
+* `os` - (Optional) Node OS.
+  * Nodes in clusters of Kubernetes `v1.13` or later support `EulerOS 2.5`.
```
Comment (lego963) [inline docs/resources/cce_node_v3.md]: mb would be better to do Note here?

File: docs/resources/cce_node_v3.md [hunk 2]
```diff
@@ -52,12 +63,6 @@ The following arguments are supported:
 
 * `annotations` - (Optional) Node annotation, key/value pair format. Changing this parameter will create a new resource.
 
-* `flavor_id` - (Required) Specifies the flavor id. Changing this parameter will create a new resource.
```
Comment (lego963) [inline docs/resources/cce_node_v3.md:55]: why did you delete this args?

**Review by lego963 — CHANGES_REQUESTED** [review]

File: docs/resources/cce_node_v3.md (hunk 1 above)
Comment (outcatcher) [reply docs/resources/cce_node_v3.md]: It's impossible to use a list in the note. Also, it's common to use a format like that, e.g `cce_cluster_v3.flavor_id`.

File: docs/resources/cce_node_v3.md (hunk 2 above)
Comment (outcatcher) [reply docs/resources/cce_node_v3.md:55]: Required args were moved to the top

Comment (lego963) [reply docs/resources/cce_node_v3.md:55]: didn't observe

**Review by lego963 — DISMISSED** [review]

**Review by lego963 — APPROVED** [review]

**Review by anton-sidelnikov — APPROVED** [review]
